### PR TITLE
Test action fixes

### PR DESCRIPTION
**Pull request by orakili**
1. Test code from custom themes as well
2. Fix output formatting
3. Skip coverage step when there are no tests

### CODE REVIEW

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -161,19 +161,21 @@ jobs:
           issue-number: ${{ github.event.pull_request.number }}
           body: |
             ### Build output
-            #### Composer Validate \`${{ steps.validate.outcome }}\`
-            #### PHP Lint \`${{ steps.lint.outcome }}\`
-            #### Docker Build \`${{ steps.build.outcome }}\`
-            #### Environment Setup \`${{ steps.docker.outcome }}\`
-            #### Site Install \`${{ steps.install.outcome }}\`
-            #### PHP Code Sniffer \`${{ steps.phpcs.outcome }}\`
```
Comment (cafuego) [inline .github/workflows/run-tests.yml:169]: Removing the backslash removes the markdown, is that the desired goal?

**Review by cafuego — APPROVED** [review]
Approved in theory.

File: .github/workflows/run-tests.yml (hunk 1 above)
Comment (orakili) [reply .github/workflows/run-tests.yml:169]: @cafuego That's not what I'm seeing in the built output in the github comments:

With the backslashes:

Screen Shot 2023-02-21 at 15 40 30

Without the backslashes:

Screen Shot 2023-02-21 at 15 40 40

The latter seems to be the expected output and that's why I removed them. Isn't that what we want?

Comment (orakili) [reply .github/workflows/run-tests.yml:169]: I see what's going on. The text was copied from [sesame](https://github.com/UN-OCHA/sesame-site/blob/d6fef2dd01a008d8219c1afdbc7b5a15e6af9934/.github/workflows/run-tests.yml#L106-L131) (or another site that adopted github actions early) but the step was changed to use a different simpler action for the comment step: `peter-evans/create-or-update-comment@v2`. With this action, the backslashes are not needed.

**Pull request merged**